src/App.jsx

```diff
@@ -1,35 +1,37 @@
-import { useState } from 'react'
-import reactLogo from './assets/react.svg'
-import viteLogo from '/vite.svg'
-import './App.css'
+import { BrowserRouter, Routes, Route, Link } from "react-router-dom";
+import Home from "./pages/Home.jsx";
+import Search from "./pages/Search.jsx";
+import Details from "./pages/Details.jsx";
 
-function App() {
-  const [count, setCount] = useState(0)
+const navStyle = {
+  display: "flex",
+  gap: 16,
+  padding: "12px 16px",
+  borderBottom: "1px solid #eee",
+  position: "sticky",
+  top: 0,
+  background: "#fff",
+  zIndex: 10,
+};
```

Comment:
separate the styles into a separate CSS file to keep the component clean and maintainable.